src/components/Blinds/Blinds.jsx

```diff
@@ -0,0 +1,58 @@
+import React, {useState} from 'react';
+import { render } from 'react-dom';
+import './style.css';
+import blindsOpen from '../../assets/blinds-open.svg';
+import blindsClosed from '../../assets/blinds-closed.svg';
+
+export const Blinds = (props) => {
+
+    let initialOpened = false;
+    if(props.blinds == 'open')
+    {
+        console.log(props.blinds);
+        initialOpened = true;
+    }
+    else 
+    {
+        initialOpened = false;
+    }
+    const [opened, setOpened] = useState(initialOpened);
```

Comment:
tohle je všechno zbytečný přepinkávání si hodnot, jde zjednodušit na
`    const [opened, setOpened] = useState(props.blinds === 'open');`
👍 (ale jinak to samozřejmě funguje)